HISTORY.rst

```diff
@@ -27,6 +27,22 @@ Bugfixes
 - Correctly refuse to send header blocks that contain neither :authority nor
   Host headers, or header blocks which contain mismatched :authority and Host
   headers, per RFC 7540 Section 8.1.2.3.
+- Correctly refuse to send header blocks with a TE header whose value is not
+  ``trailers``, per RFC 7540 Section 8.1.2.2.
+- Correctly refuse to send header blocks with connection-specific headers,
+  per RFC 7540 Section 8.1.2.2.
+
+Bugfixes (Breaking)
+~~~~~~~~~~~~~~~~~~~
```

Comment:
Heh, actually, would you like to remove this section? Some of these bug fixes are breaking anyway.